src/lib/DateInput.svelte

```diff
@@ -11,32 +11,38 @@
 
   const dispatch = createEventDispatcher<{ select: undefined }>()
 
-  const defaultDate = new Date()
+  /** Default date to use before value is assigned */
+  export let defaultDate = new Date();
```

Comment:
I don't think `defaultDate` needs to be exported, users could just set the date to `null` instead

Reply:
I think my team wants to start with a default date in the picker *without* assigning it to our value. So we can pass in a default date - that displays in the picker - and the value will remain null until the user actively selects a date (or types one in). De-coupling these lets the caller decide how they want to use it. I'll try to get this working properly in the demo so you can see what i mean (It's still a WIP)

Reply:
Ah, I see, maybe `preselectedDate` would be a better name. Think you could add that in another PR though, and keep this one just for allowing `null` values?

Reply:
...per the question, let me think about separating the null support from the value-default decoupling